### PR TITLE
Airtable xts7 - feedback needed

### PR DESCRIPTION
**Pull request by xts7**
Please review my code, thx

### CODE REVIEW

File: pages/cars.vue [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<template>
+ <h1> {{ data.cars }} </h1>
+</template>
+
+<script setup>
+const { data } = await useFetch('/api/cars')
```
Comment (razbakov) [inline pages/cars.vue:6]: how this suppose to work?

**Review by razbakov — CHANGES_REQUESTED** [review]

File: pages/cars.vue (hunk 1 above)
Comment (razbakov) [reply pages/cars.vue:6]: It probably won't work on Netlify, cuz it's using Nuxt Server.

Comment (xts7) [reply pages/cars.vue:6]: oh ok, i will try to do it on the page then

File: server/api/cars.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    const apiUrl = `https://api.airtable.com/v0/app5FuD1YnKgDpJJz/Cars?api_key=${apiKey}`
+    const cars = '';
+    let data = {data: [{data: ""}]}
+
+    if(cars){
+        data = await $fetch (apiUrl)
+    }
+    res.write(JSON.stringify(data));
+    res.end();
+}
```
Comment (razbakov) [inline server/api/cars.ts:15]: Please always add a blank line in the end of each file

**Pull request closed without merging**